_docs/_latest/concepts/real-time-enforcer.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Forseti Real Time Enforcer
+order: 005
+---
+
+# {{ page.title }}
+
+> _This is a beta release of Forseti Real Time Enforcer. This product might be changed in backward-incompatible ways 
```

Comment:
nit: ditto on dropping `Forseti` when describing this without the context of our website. Can you please change it everywhere?

Reply:
Updated.